tools/windows.py

```diff
@@ -29,6 +30,9 @@ def generate(env):
             env.Append(CCFLAGS=["/Z7", "/Od", "/EHsc", "/D_DEBUG", "/MDd"])
         elif env["target"] == "release":
             env.Append(CCFLAGS=["/O2", "/EHsc", "/DNDEBUG", "/MD"])
+        if env["use_clang_cl"]:
+            env["CC"]="clang-cl"
+            env["CXX"]="clang-cl"
```

Comment:
missing space between ` = `

Reply:
Ah, sorry, I guessed that was a typo in the suggested change - so the right style is to have one space on the left of the `=` and no space on the right? I'll update after work.

Reply:
I *think* the right format is actually space on both sides of the operator, which makes more sense to me, so hopefully what I just did was correct.